src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Find.cs

```diff
@@ -75,7 +75,7 @@ public override bool CheckInvocation(TexlBinding binding, TexlNode[] args, DType
             var type1 = argTypes[1];
 
             // Arg0 should be either a string or a column of strings.
-            if (type0.IsTable)
+            if (type0.IsTable && type0.Kind != DKind.ObjNull)
```

Comment:
Why are we making changes to the front-end for this function?

Reply:
yep, bad idea. ignored the fact that JS tests for Find operate post-frontend.